lib/cocoapods_packing_cubes.rb

```diff
@@ -1,59 +1,19 @@
 # frozen_string_literal: true
 
 module CocoaPodsPackingCubes
-  if defined?(::Pod::Target::Type)
-    Type = ::Pod::Target::Type
-    NATIVE_TYPE_SUPPORT = true
-  else
-    class Type
-      def initialize(linkage: :static, packaging: :library)
-        @linkage = linkage
-        @packaging = packaging
-
-        case [linkage, packaging]
-        when %i[static library], %i[static framework], %i[dynamic framework]
-          # ok
-          nil
-        else
-          raise ::Pod::Informative, "Unsupported target build type: #{inspect}"
-        end
-      end
-
-      attr_reader :linkage, :packaging
-      protected :linkage, :packaging
-
-      def ==(other)
-        linkage == other.linkage &&
-          packaging == other.packaging
-      end
-      alias eql? ==
-
-      def hash
-        linkage.hash ^ packaging.hash
-      end
-
-      %i[static dynamic].each_with_index do |linkage, index|
-        define_method("#{linkage}?") { linkage == @linkage }
-        %i[library framework].each do |packaging|
-          if index.zero?
-            define_method("#{packaging}?") { packaging == @packaging }
-          end
-
-          define_method("#{linkage}_#{packaging}?") do
-            linkage == @linkage && packaging == @packaging
-          end
-        end
-      end
+  def self.warn_for_outdated_cocoapods
+    @warn_for_outdated_cocoapods ||= begin
+      Pod::UI.warn '[cocoapods-packing-cubes] Pod::Target::BuildType must be defined, either:',
+                   ['Update to CocoaPods master or 1.6.0+', 'Downgrade to packing cubes < 0.3']
```

Comment:
Add this version dependency to the `gemspec`?

Reply:
can't, because then cocoapods could never end up depending on the plugin